plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -10,5 +10,6 @@ Require-Bundle: org.eclipse.ui,
  javax.servlet;bundle-version="2.5.0",
  javax.servlet.jsp;bundle-version="2.2.0",
  org.eclipse.core.resources,
- org.eclipse.ui.ide
+ org.eclipse.ui.ide,
+ org.eclipse.core.runtime
```

Comment:
Use `org.eclipse.equinox.common` instead.  Try to avoid `.core.runtime` as it brings in all sorts of dependencies for historical reasons.